src/ElectricPokemon.java

```diff
@@ -0,0 +1,115 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+    private final String type = "Electric";
+    private final List<String> attacks = Arrays.asList("Thunder Punch", "Electro Ball", "Thunder", "Volt Tackle");;
+
+    public ElectricPokemon(String name, int level, int hp, String food, String sound){
+        super(name, level, hp, food, sound);
+    }
+
+    void thunderPunch(Pokemon name, Pokemon enemy){
+        int damage = 20;
+        float modifier;
+        System.out.println(name + " uses Thunder Punch on " + enemy + "!");
+
+        switch (enemy.getType()) {
+            case "Electric":
+                modifier = 0.25f;
+                break;
+            case "Fire":
+                modifier = 0.5f;
+                break;
+            case "Water":
+                modifier = 2.0f;
+                break;
+            default:
```

Comment:
Default is hier misschien neit handig, want je switched op een String, dus het kan in principe ook "banaan"  zijn. Zou je switchen op een Enum met 4 waardes, dan is het prima om de vierde waarde als default te zetten of anders de default weg te laten.